source/mainnet/net/guides/run-node-aws.rst

```diff
@@ -0,0 +1,165 @@
+.. _Discord: https://discord.gg/xWmQ5tp
+
+.. _run-a-node-aws:
+
+=================
+Run a node on AWS
```

Comment:
**Suggestion (non-blocking):** Introduce the `AWS` abbreviation to help the reader, but also improve search.
```suggestion
Run a node on Amazon Web Services (AWS)
```